### PR TITLE
spreadshirt.com

### PR DESCRIPTION
**Pull request by gork7777**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

### Enter the issue address
https://github.com/AdguardTeam/AdguardFilters/issues/123321

1. Your comment

There are many subdomains for the `spreadshirt` domain

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -123,6 +123,9 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+esprit.de#$#.mfp-wrap { display: none !important; }
+esprit.de#$#.mfp-bg { display: none !important; }
+esprit.de#$#html { overflow: auto !important; padding-right: 0 !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:128]: How to reproduce it? Please send a screenshot with code of the element.

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -2448,7 +2450,7 @@ markmanson.net#$##exit-modal { display: none!important; }
 markmanson.net#$#body { overflow: visible!important; }
 winaero.com###dwpb
 winaero.com##.dwpb-close
-spreadshirt.com##.promo-popup
+spreadshirt.*##.promo-popup
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:2453]: Yep, I can see that.

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (gork7777) [reply AnnoyancesFilter/sections/popups.txt:128]: @zloyden Well, I can't reproduce this time.

Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt:128]: Please remove these rules except your fix for `spreadshirt`.

**Review by zloyden — APPROVED** [review]

**Pull request merged**